app/modules/user/payout_schedule.rb

```diff
@@ -81,6 +82,8 @@ def last_friday_of_quarter(date)
 
     def get_initial_payout_date(date)
       case payout_frequency
+      # Daily payouts are handled separately, so this date is a fallback, for any amount not able to be paid instantly
```

Comment:
When daily frequency is set, we don't use the `next_payout_date` logic, since it is so closely tied to the previously weekly system.

This is confusing, and something we should fix (perhaps with better naming) in the future. Technically, the payout model is no longer simple enough to just have *one* next payout period.

We use the existing weekly (standard) payout as a fallback for any balance that was not paid out by the daily instant payout job.

So technically, a user may now have a next (daily, instant) payout period, *and* a next (weekly, standard, fallback) payout period.

Reply:
With @hchhabra's latest changes, `next_payout_date` is used after all, resolving.